_posts/elements/2015-07-01-grid.md

```diff
@@ -0,0 +1,710 @@
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+
+            <div class="col col-2">
+                <span class="example-label">col-2</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+
+            <div class="col col-3">
+                <span class="example-label">col-3</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-4">
+                <span class="example-label">col-4</span>
+            </div>
+
+            <div class="col col-4">
+                <span class="example-label">col-4</span>
+            </div>
+
+            <div class="col col-4">
+                <span class="example-label">col-4</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-12">
+                <span class="example-label">col-12</span>
+            </div>
+        </div>
+
+        <div class="row">
+            <div class="col col-2 pre-5 post-5">
+                <span class="example-label">col-2</span>
+                <span class="example-label">pre-5</span>
+                <span class="example-label">post-5</span>
+            </div>
+
+            <div class="col col-4 pre-4 post-4">
+                <span class="example-label">col-4</span>
+                <span class="example-label">pre-4</span>
+                <span class="example-label">post-4</span>
+            </div>
+
+            <div class="col col-6 pre-3 post-3">
+                <span class="example-label">col-6</span>
+                <span class="example-label">pre-3</span>
+                <span class="example-label">post-3</span>
+            </div>
+
+            <div class="col col-8 pre-2 post-2">
+                <span class="example-label">col-8</span>
+                <span class="example-label">pre-2</span>
+                <span class="example-label">post-2</span>
+            </div>
+
+            <div class="col col-10 pre-1 post-1">
+                <span class="example-label">col-10</span>
+                <span class="example-label">pre-1</span>
+                <span class="example-label">post-1</span>
+            </div>
+        </div>
+    </div>
+</div>
+
+<h3 class="hd-6 example-set-hd">Utility Class-based Nested Grid</h3>
```

Comment:
@frrrances, there's a current issue this example shows. Using the DOM-based method of grid use (throwing classes on things), Susy has no sense of context (the parent's column width) and thus falls back to assuming a whole 12 columns when calculating column width. 

* Here's a good read up on Susy context - http://www.zell-weekeat.com/context-with-susy/.
* This is easily fixed when using the Sass method through either the ``span(2 of 8)`` syntax or using the ``@include nested(8)`` mixin (http://susy.readthedocs.org/en/latest/toolkit/#nested-context).

So we have a few options:

1. **Give Susy the context it needs by adding more classes to the DOM** (see the below example). This means adding more classes to our Sass/CSS (including context-based classes for each responsive size too :/)

```html
<div class="grid-container grid-manual">
        <div class="row">
            <div class="col col-8">
                <div class="row">
                    <div class="col col-2of8">
                        ...
                    </div>

                    <div class="col col-2of8">
                        ...
                    </div>

                    <div class="col col-2of8">
                        ...
                    </div>

                    <div class="col col-2of8">
                        ...
                    </div>
                </div>

                <div class="row">
                    <div class="col col-4of8">
                        ...
                    </div>

                    <div class="col col-4of8">
                        ...
                    </div>
                </div>

                <div class="row">
                    <div class="col col-2of8">
                        ...
                    </div>

                    <div class="col col-2of8">
                        ...
                    </div>

                    <div class="col col-2of8">
                        ...
                    </div>
                </div>
            </div>

            <div class="col col-4">
                <div class="row">
                    <div class="col col-2of4">
                        ...
                    </div>

                    <div class="col col-2of4">
                        ...
                    </div>
                </div>

                <div class="row">
                    <div class="col col-4of4">
                        ...
                    </div>
                </div>
            </div>
        </div>
    </div>
```

2. **We just don't allow nesting through the DOM-based method** - this seems like it may cause headaches with Non-FED folks trying to add more layout to an existing manual-based layout

- - -

What do you think?